Platform/TigerlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -480,17 +479,20 @@ UpdateFspConfig (
     Fspmcfg->CpuPcieRpEnableMask           = 0;
   }
 
-  if (FeaturesCfgData->Features.LowPowerS0Idle == 1)
-  {
-    Fspmcfg->TcssXdciEn=0;
-    Fspmcfg->TcssXhciEn=0;
-    Fspmcfg->TcssDma0En=0;
-    Fspmcfg->TcssDma1En=0;
-    Fspmcfg->TcssItbtPcie0En=0;
-    Fspmcfg->TcssItbtPcie1En=0;
-    Fspmcfg->TcssItbtPcie2En=0;
-    Fspmcfg->TcssItbtPcie3En=0;
-    Fspmcfg->DmiAspmCtrl            = 2;// ASPM configuration on the CPU side of the DMI/OPI Link
+  FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag(CDATA_FEATURES_TAG);
+  if (FeaturesCfgData != NULL) {
+    if (FeaturesCfgData->Features.LowPowerS0Idle == 1)
+    {

```

Comment:
Better putting them to one line as below to make the changes minimal, Same to other places.
if ((FeaturesCfgData != NULL) && (FeaturesCfgData->Features.LowPowerS0Idle == 1)) {

Reply:
Done, thanks for the review.